src/csharp/Microsoft.Spark.UnitTest/UdfSerDeTests.cs

```diff
@@ -16,7 +16,7 @@ public class UdfSerDeTests
         [Serializable]
         private class TestClass
```

Comment:
Can you update the documentation to include this feature? This is a pretty cool one so having a good e2e example would be great so I can tell everyone.

Reply:
Yes, I plan to do that as a part of #147 or a separate PR.

Reply:
Got it, thanks!